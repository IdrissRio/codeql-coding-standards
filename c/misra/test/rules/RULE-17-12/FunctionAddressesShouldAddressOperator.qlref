rules/RULE-17-12/FunctionAddressesShouldAddressOperator.ql