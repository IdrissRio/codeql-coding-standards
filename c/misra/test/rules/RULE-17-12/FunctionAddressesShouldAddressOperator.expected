| test.c:14:25:14:29 | func2 | The address of function func2 is taken without the & operator. |
| test.c:15:25:15:29 | func3 | The address of function func3 is taken without the & operator. |
| test.c:21:12:21:16 | func1 | The address of function func1 is taken without the & operator. |
| test.c:38:3:38:7 | func1 | The address of function func1 is taken without the & operator. |
| test.c:39:3:39:7 | func2 | The address of function func2 is taken without the & operator. |
| test.c:57:13:57:17 | func1 | The address of function func1 is taken without the & operator. |
| test.c:58:21:58:25 | func2 | The address of function func2 is taken without the & operator. |
| test.c:59:13:59:17 | func1 | The address of function func1 is taken without the & operator. |
| test.c:59:20:59:24 | func2 | The address of function func2 is taken without the & operator. |
| test.c:67:11:67:15 | func1 | The address of function func1 is taken without the & operator. |
| test.c:68:12:68:16 | func1 | The address of function func1 is taken without the & operator. |
| test.c:69:12:69:16 | func1 | The address of function func1 is taken without the & operator. |
| test.c:71:18:71:22 | func1 | The address of function func1 is taken without the & operator. |
